spec/features/user_sees_jockeys_total_winnings_spec.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../spec_helper'
+
+RSpec.describe "when a user visits a jockey's page" do
+  it "they can see the jockey's total winnings" do
+    Breed.create(name: "Palomino")
+    joe = Jockey.create(name: "Joe")
+
+    joe.horses.create(name: "Megan", age: 5, total_winnings: 3, breed_id: 1)
+    joe.horses.create(name: "Phil", age: 4, total_winnings: 5, breed_id: 1)
+    joe.horses.create(name: "Flippy", age: 6, total_winnings: 4, breed_id: 1)
+    joe.horses.create(name: "Muffin", age: 22, total_winnings: 1, breed_id: 1)
+
+    visit("/jockeys/#{joe.id}")
+    expect(page).to have_content(13)
```

Comment:
Think this works here, but this is the kind of place where you might want to scope this down to a specific css block or include a little more information in your expectation. It feels like there's a decent chance of `13` being *somewhere* on the page, giving us a false positive on a passing test. I could see this as either of the following two options:

```ruby
# Option 1
# This feels a little gross because it seems like we're adding an `id` just for our test
within('#total_winnings') do
  expect(page).to have_content(13)
end

# Option 2
# Personally prefer this, but may not always be possible/preferable.
expect(page).to have_content("Total Winnings: 13")

```